gradle.properties

```diff
@@ -17,11 +17,11 @@ POM_NAME=MarvelApiClient
 POM_ARTIFACT_ID=MarvelApiClient
```

Comment:
@pedrovgs @Serchinastico do you like that I change the name for marvel-api-client ??